internal/services/order/order.go

```diff
@@ -1,13 +1,212 @@
 package order
 
-func BuyOrder(BReq BuyRequest) (bRes BuyResponse, err error) {
-	return bRes, err
+import (
+	"Tradeasy/config"
+	"Tradeasy/internal/model"
+	"Tradeasy/internal/services/stock_exchange"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"github.com/google/uuid"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+func BuyOrder(bReq BuyRequest) (bRes stock_exchange.OrderResponse, err error) {
+	var checkLtp model.StocksFeed
```

Comment:
When initialising model objects, better to use similar names to avoid any confusion.